src/SingleCellAnnotator.imjoy.html

```diff
@@ -468,7 +467,7 @@
       const locs = {}
       if(this.current_image_data.location_labels){
         for(let k in this.current_image_data.location_labels){
-          if(!this.LIMS_VALUES.includes(k)){
+          if((!this.LIMS_VALUES.includes(k))&&(k !== 'Microtubule organizing center')){
```

Comment:
We should perhaps fix this from the backend?

Reply:
Right, It is better to do that from backend.

Reply:
now the moc is resolved from the backend, thus ignored from frontend.